### DISCUSSION TITLE
Highlighting queries / articles in scans

### DISCUSSION THREAD

**Discussion by JeltevanBoheemen** [UUDigitalHumanitieslab/I-analyzer#1421] [Ideas]
See https://github.com/VadimDez/ng2-pdf-viewer for implementation of query highlighting using this package. This has the side effect of resetting the page to the document page after a few seconds. This is probably due to the underlying PDFJs package that scrolls to the first encountered match. 

Have to decide if we use another package for pdf altogether, try to solve the bug, or implement our own highlighting.

**Comment by BeritJanssen** [comment]
I renamed the issue to something more general. As far as I can see, we can highlight in the following three corpora:
- Times (through alto)
- Dutch Annual Reports (pdf / alto)
- Guardian (pdf)

I added a time estimate, which is based on getting to work both the alto and the pdf mechanism.

We cannot provide highlighting for all corpora, so I think the time investment for highlighting in these three corpora may not be warranted at the moment, unless stakeholders tell us they really, really need this feature. Putting on hold.
